### PR TITLE
Add analytics sample

### PR DESCRIPTION
**Pull request by sebastianbenz**
+ simple ping API to store generic events
+ simple view showing count of all events

//cc @jmadler

### CODE REVIEW

File: models/amp-analytics.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  var data = EVENTS[account];
+  if (!data) {
+    data = {};
+    EVENTS[account] = data;
+  }
+  var eventCount = data[event];
+  if (!eventCount) {
+    eventCount = 0;
+  }
+  return data[event] = eventCount + 1;
```
Comment (andreban) [inline models/amp-analytics.js]: I don't know much about how Node works when handling multiple requests. Can we have a race condition on this? If so, i think we can add a comment, just as a note that the counter may not be precise.

Comment (sebastianbenz) [inline models/amp-analytics.js]: Good point, but I think we don't have a problem here as Node is single threaded.

**Comment by andreban** [comment]
LGTM

File: models/amp-analytics.js (hunk 1 above)
Comment (andreban) [inline models/amp-analytics.js]: In this case, all good

**Pull request merged**